### DISCUSSION TITLE
OTEL transaction naming | Identity transaction per protocol://host:port/path

### DISCUSSION THREAD

**Discussion by ludovic-pourrat** [Kong/kong#11761] [Ideas and feature requests]
As of today when enabling the OTEL plugin into Kong, all the transactions are named statically "kong" where usually the OTEL agents pick up a transaction identification that reflects better the transactions. In the case of Kong the transition should be something related to the route exposed.

This feature request aim to provide an addition property to named the OTEL transaction differently based on the route definition to follow a common pattern in the OTEL, e.g. `protocol://host:port/path`.

I will provide a pull request to illustrate this.

Feedback, and advices are welcome !

**Comment by samugi** [comment]
Closing this one to keep the conversation in one place (#11762 )
